velox/common/memory/MemoryUsageTracker.h

```diff
@@ -88,20 +104,21 @@ class MemoryUsageTracker
     return create(nullptr, maxMemory);
   }
 
-  virtual ~MemoryUsageTracker() = default;
+  // TODO: add check all the reserved and allocated bytes must have been freed.
```

Comment:
Any particular reason we cannot address this TODO now? I remember you have added a similar check in the past.

Reply:
I want to turn on this check in a separate followup PR as we did before.

Reply:
Added.